lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -2991,6 +2992,7 @@ class SwiftDWARFImporterDelegate : public swift::DWARFImporterDelegate {
       // described in DWARF.
       return true;
     }
+    return false;
```

Comment:
so msvc doesn't consider the switch exhaustive? should this be an `llvm_unreachable`?

Reply:
Yes, MSVC doesn't consider the paths completely covered.  I can convert both of them to `llvm_unreachable`.